be/src/storage/vectorized/compaction.h

```diff
@@ -18,6 +18,15 @@ namespace starrocks::vectorized {
 
 class DataDir;
 
+enum CompactionAlgorithm {
+    // compaction by all columns together.
+    HORIZONTAL = 0,
```

Comment:
```suggestion
    kHorizontal = 0,
```
https://google.github.io/styleguide/cppguide.html#Enumerator_Names